### PR TITLE
Update 09-seshakus.md

### PR DESCRIPTION
**Pull request by seshakus**

### CODE REVIEW

File: _slides/09-seshakus.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I am Sesha Kintali
```
Comment (ladkaushik) [inline _slides/09-seshakus.md:7]: This is kaushik, changed your code abd add comment

**Review by ladkaushik — CHANGES_REQUESTED** [review]

**Review by seshakus — COMMENTED** [review]
Good

**Review by seshakus — COMMENTED** [review]
Good

File: _slides/09-seshakus.md (hunk 1 above)
Comment (seshakus) [reply _slides/09-seshakus.md:7]: Good

**Review by seshakus — COMMENTED** [review]
Good

**Comment by seshakus** [comment]
..

**Review by Chocrates — APPROVED** [review]
🦑

**Review by Chocrates — DISMISSED** [review]

File: _slides/09-seshakus.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (seshakus) [inline _slides/09-seshakus.md:6]: Ok

**Comment by ladkaushik** [comment]
thanks

**Pull request closed without merging**